etc/config/ada.amazon.properties

```diff
@@ -65,25 +65,27 @@ compiler.gnats390x1120.semver=11.2.0
 ################################
 # GNAT for ppc
 group.gnatppcs.compilers=&gnatppc:&gnatppc64:&gnatppc64le
-group.gnatppcs.groupName=GNAT power
 group.gnatppcs.instructionSet=ppc
 group.gnatppcs.isSemVer=true
 group.gnatppcs.supportsBinary=false
 group.gnatppcs.supportsExecute=false
 
 ## POWER
+group.gnatppc.groupName=GNAT power
```

Comment:
I'm wondering if it makes sense to have any of these "GNAT" prefixes, we only have gnat compilers.

Reply:
I was thinking about adding the HAC compiler later (https://hacadacompiler.sourceforge.io/). But sure, currently, there's only  GNAT, so removing it from the group names makes complete sense. Will do that :+1: